doc/varnish-configuration.rst

```diff
@@ -36,111 +36,162 @@ will be used throughout the Varnish examples on this page.
 Provided Vcl Subroutines
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-In order to ease configuration we provide a set of vcl subroutines in the resources/config directory.
-These can be directly included into ``your_varnish.vcl`` and the needed subroutines called
-from the respective vcl_* subroutines.
+In order to ease configuration we provide a set of vcl subroutines in the
+``resources/config`` directory. These can be included from your main Varnish
+configuration file, typically ``default.vcl``. Then you need to make your
+``vcl_*`` subroutines call the ``fos_*`` routines.
 
-.. important::
-    When including one of the provided vcl you need to call all the defined subroutines
-    or your configuration will not be valid.
+.. tip::
+    When including one of the provided vcl, you need to call all the defined
+    subroutines or your configuration will not be valid.
 
-    See the respective sections below on how to configure usage of each of the provided vcl's.
+    See the respective sections below on how to configure usage of each of the
+    provided vcl's.
```

Comment:
yeah, i normalized to lowercase... fixed to uppercase now, preferring that as well.